tests/__snapshots__/ProductSettingsMigrationTest__it_migrates_pre_v5.0.0_product_settings_with_data_set_simple_product_with_all_settings__1.json

```diff
@@ -40,8 +40,8 @@
         "exportOnlyRecipient": -1,
         "exportReturn": -1,
         "exportSignature": -1,
-        "fitInDigitalStamp": 7,
-        "fitInMailbox": 7,
+        "fitInDigitalStamp": 0,
+        "fitInMailbox": 1000,
```

Comment:
waarom is deze ineens anders? dit lijkt me niet correct

Reply:
Klopt, ik heb de MockWcCart bijgewerkt zodat hij meer lijkt op de echte cart. Dit heeft het probleem opgelost.